_posts/2022-02-25-roadmap-proposal.md

```diff
@@ -31,17 +31,16 @@ The team wants to produce minor-version releases on a "release-train" schedule:
 
 ### Proposed schedule
 
-Taking the above into consideration, here’s how I think the majors and minors would lay out on the calendar.  If you’re planning on submitting a code change, note the corresponding feature freeze dates (the date when no more changes can be added so the release can be tested). 
+Taking the above into consideration, here’s how I think the majors and minors would lay out on the calendar.  If you’re planning on submitting a code change, note the corresponding feature freeze dates (the date when no more changes can be added so the release can be tested).
 
 |Version	|Feature Freeze	|Release Date	|
 |-------------	|-------------	|-------------	|
 |1.3.0	|March 9th	|March 15th	|
-|2.0.0 Preview Release	|March 21st	|March 31st	|
-|1.4.0	|April 20th	|April 26th	|
+|2.0.0 RC	|Core: March 21st <br>Plugins and Clients: April 18th	|April 26th	|
```

Comment:
This is not great visually. Ends up looking messy and confusing.

What if you did something like this? I think if you add another row below it with a blank cell for `Version` and `Release Date` you could achieve this.


<img width="617" alt="Screen Shot 2022-03-23 at 3 50 26 PM" src="https://user-images.githubusercontent.com/1152927/159801884-f7de651b-1b2a-4794-88b7-c6d5fc248ff0.png">